### ISSUE TITLE
Missing documentation ("How to use" section in README.md)

### ISSUE THREAD

**Issue by MakisH** [ajaust/parameter-study-precice#1]
I know, it's probably coming soon, but just as a reminder to make this tool usable by the preCICE community. :sweat_smile:

**Comment by ajaust** [comment]
Good point. I will have to check myself how to use it and whether it works at the moment... 😅
